projects/dotcms-field-elements/src/components/dot-key-value/key-value-table/key-value-table.tsx

```diff
@@ -0,0 +1,43 @@
+import { Component, Prop, Event, EventEmitter } from '@stencil/core';
+import { KeyValue } from '../../../models';
+
+@Component({
+    tag: 'key-value-table'
+})
+export class DotTextfieldComponent {
+    @Prop() keyValues: KeyValue[] = [];
+    @Prop() disabled = false;
+
+    @Event() deleteKeyValue: EventEmitter;
+
+    render() {
+        return (
+            <table>
+                <tbody>
+                    {this.keyValues.map((item, index) => {
+                        return (
+                            <tr>
+                                <td>
+                                    <button
+                                        type="button"
+                                        disabled={this.disabled || null}
+                                        onClick={() => this.deleteKey(index)}
+                                        class="dot-key-value__delete__button"
+                                    >
+                                        <i class="dot-key-value__delete-button__icon" />
```

Comment:
For now... use a label "Delete" and the user can set it, with the `dot-key-value__delete__button` the devs can set an icon and hide the label.